account_followup/models/res_partner.py

```diff
@@ -0,0 +1,447 @@
+        # _() looks for the lang in local variable 'context'.
+        # Set the language to use = the partner language
+        followup_table = ''
+        if partner.unreconciled_aml_ids:
+            company = self.with_context(lang=partner.lang).env.user.company_id
+            current_date = fields.date.context_today(
+                self.with_context(lang=partner.lang))
+            rml_parse = \
+                account_followup_print.report_rappel("followup_rml_parser")
+            final_res = rml_parse._lines_get_with_partner(partner, company.id)
+
+            for currency_dict in final_res:
+                currency = currency_dict.get(
+                    'line', [
+                        {'currency_id': company.currency_id}
+                    ])[0]['currency_id']
+                followup_table += '''
+                <table border="2" width=100%%>
+                <tr>
+                    <td>''' + _("Invoice Date") + '''</td>
+                    <td>''' + _("Description") + '''</td>
+                    <td>''' + _("Reference") + '''</td>
+                    <td>''' + _("Due Date") + '''</td>
+                    <td>''' + _("Amount") + " (%s)" % \
+                                            currency.symbol + '''</td>
+                    <td>''' + _("Lit.") + '''</td>
+                </tr>
+                '''
+                total = 0
+                for aml in currency_dict['line']:
+                    block = aml['blocked'] and 'X' or ' '
+                    total += aml['balance']
+                    strbegin = "<TD>"
+                    strend = "</TD>"
+                    date = aml['date_maturity'] or aml['date']
+                    if date <= current_date and aml['balance'] > 0:
+                        strbegin = "<TD><B>"
+                        strend = "</B></TD>"
+                    followup_table += "<TR>" + strbegin + str(aml['date']) + \
+                                      strend + strbegin + aml['name'] + \
+                                      strend + strbegin + (aml['ref'] or '') \
+                                      + strend + strbegin + str(date) + \
+                                      strend + strbegin + str(aml['balance']) \
+                                      + strend + strbegin + block + strend \
+                                      + "</TR>"
+
+                total = reduce(lambda x, y: x+y['balance'],
+                               currency_dict['line'], 0.00)
+
+                total = rml_parse.formatLang(total, dp='Account',
+                                             currency_obj=currency)
+                followup_table += '''<tr> </tr>
+                                </table>
+                                <center>''' + _("Amount due") \
+                                  + ''' : %s </center>''' % total
+        return followup_table
+
+    @api.multi
+    def write(self, values):
+        if values.get("payment_responsible_id", False):
```

Comment:
Don't indent too much.
```
if not values.get(...):
    return super(...)
<do stuff>
return super(...)
```
It's better to repeat an instruction in these cases than to be stretched by a new indentation level.